mqtt/mqttd/src/broker/mod.rs

```diff
@@ -18,10 +18,15 @@ use mqtt_broker::{
 };
 use mqtt_broker_core::auth::Authorizer;
 
+const MQTTD_STATE_DIR: &str = "/tmp/mqttd";
```

Comment:
unfortunately, it's not THAT simple :( also it will break perf tests.
it should be handled by the config section which was under refactoring now.